src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/LocalDBTest/LocalDBTest.cs

```diff
@@ -155,6 +185,12 @@ private static string ExecuteLocalDBCommandProcess(string filename, string argum
                 RedirectStandardError = true,
             };
             string[] lines = Process.Start(sInfo).StandardOutput.ReadToEnd().Split(new string[] { Environment.NewLine }, StringSplitOptions.None);
```

Comment:
Please verify the expected result by running the command. Like assessing the number of returned lines.

Reply:
Added asserts in if/else as they can vary